app/views/shared/code/_extract_method_6.html.erb

```diff
@@ -0,0 +1,21 @@
+def print_owing
+  outstanding = 0.0
+
+  print_banner
+
+  # calculate outstanding
+  @orders.each do |order|
+    outstanding += order.amount
+  end
+
+  # print details
+  puts "name: #{@name}"
+  puts "amount: #{outstanding}"
+end
+
+def print_banner
+  # print banner
+  puts "*************************"
+  puts "***** Customer Owes *****"
+  puts "*************************"
+end
```

Comment:
If using sublime, add this to your user preferences: ` { "ensure_newline_at_eof_on_save": true }`
 
If using something else, configure accordingly.